### ISSUE TITLE
Show how many versions are supported in a browser

### ISSUE THREAD

**Issue by mirisuzanne** [oddbird/browser-support#1]
This would be like showing the browser version where support was added, but I think it might be nicer to know _how many_ versions it's been since support was added?

**Comment by jensgro** [comment]
Good idea and very usable for devs and marketing. But how would you count? In fact we the list should contain each andery subversion. 
With iPadOS having a direct connection between OS and Safari, there should be Safari 18.1.4 in the list, not only 18. Correct?

How would you communicate? "3 major version, 15 subversions ..."

And there should be a trustful source with up-to-date data of the browser versions. Each loading of the component would have  to look for the daily data.

Did I extrapolate correct?

**Comment by mirisuzanne** [comment]
Yeah, it might not actually be reasonable to do. I wonder about using a date instead. That still gives a sense of how long it's been in a browser, without getting into the different release cycles and versioning systems.

I was thinking at one point about a progress bar for each browser. Basically the overall progress indicator should hit 'widely available' at the same time as the final browser does.

**Comment by jensgro** [comment]
Yes, date would be easier to grasp. And it could be a motivation to research further. In my opinion there would be no need for this webcomponent to answer all questions. It should only give the most important informations fast.

**Comment by jensgro** [comment]
I hope you don't mind me brainstorming with you :-)
